content/blog/2017/10/linux-patching-management-with-ansible.md

```diff
@@ -0,0 +1,524 @@
++++
+author = "Brian Alcorn"
+author_url = "https://github.com/balcornc"
+categories = ["Brian Alcorn", "Encore"]
+date = "2017-10-31"
+description = "Centralized Linux Patching Activity Management for Consistency, Compliance, and Scalability"
+featured = ""
```

Comment:
If you don't have a picture, please remove the "feature*" metadata otherwise the blog doesn't render properly in Safari